### PR TITLE
八小时之歌

### PR DESCRIPTION
**Pull request by ujmkiolp**
The song “Eight Hours” was first published in musical form in 1878. The song’s lyrics were written by I.G. Blanchard, and the music was written by Rev. Jesse H. Jones.

【中英字幕】八小时之歌 Eight hours（非完整版）
https://www.bilibili.com/video/av39328444/

### CODE REVIEW

**Comment by maillengxue** [comment]
Awesome!

**Comment by whatPlane** [comment]
这个很好

**Comment by zxkjack123** [comment]
I found a longer version of the 8-hour song here:
http://www.contemplator.com/america/eighthour.html
The time is 4:43, but without human sounds, just rhythm.

File: externals/music.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+奋勇前进，凯旋而归。  
+让我们一起来高呼，  
+呼声响遍人间和天上：  
+（合唱）八小时工作，八小时休息，  
+　　　　八小时归自己！  
+　　　　八小时工作，八小时休息，  
+　　　　八小时归自己！  
+
+### 链接
+[请问《八小时之歌》英文原版是什么样的？](https://www.zhihu.com/question/65527260)
```
Comment (zxkjack123) [inline externals/music.md]:
```suggestion
[《八小时之歌》英文原版](http://www.contemplator.com/america/eighthour.html)  
```

**Comment by zcylovezcy** [comment]
越高越大 国内网都封了这儿啊

**Pull request merged**